llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/insertelement.ll

```diff
@@ -0,0 +1,1941 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
```

Comment:
Are we only testing inserting into element 0?

Reply:
And only i32. Should try with some i64s and non-constants

Reply:
Updated.